src/components/atoms/Modal/Modal.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import './Modal.css'
+
+type LayerProps = {
+  children: React.ReactNode
+  isOpen: boolean
+}
+
+const Modal: React.FC<LayerProps> = ({ children, isOpen }) => {
```

Comment:
리액트에 PropsWithChildren 이라는 타입을 아시나요?
귀찮은 children을 명시안해줘도 돼요.

```suggestion
import React, type { PropsWithChildren } from 'react'
import './Modal.css'

type LayerProps = {
  isOpen: boolean
}

const Modal: React.FC<PropsWithChildren<LayerProps>> = ({ children, isOpen }) => {
```